tx-pool/src/service.rs

```diff
@@ -955,6 +978,28 @@ impl TxPoolService {
         }
     }
 
+    /// Get Live Cell Status
```

Comment:
For an outpoint, if there is already a pending transaction consuming the cell, whether it's defined as `live cell`?
From the code, it is.